### PR TITLE
use the right properties from resume data json

### PR DESCRIPTION
**Pull request by sachinkumar579**
The existing basic property is used from resume data json 

Fixes # (issue)

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/modules/builder/editor/modules/basic/BasicLayout.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import BasicPanel from './components/BasicPanel';
 const BasicLayout = () => {
   const [activeTab, setActiveTab] = React.useState(0);
   const { set: onChangeText, values: basicTabs } = useBasicDetails();
+  const tabTitles = ['Contacts', 'Links', 'About'];
```
Comment (siva-kannan3) [inline src/modules/builder/editor/modules/basic/BasicLayout.tsx:9]: move this constant outside of the function

File: src/modules/builder/editor/modules/basic/components/BasicHeader.tsx [hunk 2]
```diff
@@ -5,18 +5,18 @@ import Tab from '@mui/material/Tab';
 const BasicHeader = ({
   activeTab,
   changeActiveTab,
-  basicTabs,
+  tabTitles,
 }: {
   activeTab: number;
   changeActiveTab: (event: React.SyntheticEvent, activeTab: number) => void;
-  basicTabs: any;
+  tabTitles: any;
```
Comment (siva-kannan3) [inline src/modules/builder/editor/modules/basic/components/BasicHeader.tsx:12]:
```suggestion
  tabTitles: string[];
```

File: src/modules/builder/editor/modules/basic/components/BasicHeader.tsx [hunk 3]
```diff
@@ -5,18 +5,18 @@ import Tab from '@mui/material/Tab';
...
 }) => {
   return (
     <Tabs value={activeTab} onChange={changeActiveTab} variant="fullWidth">
-      {basicTabs.map((tab: any, index: number) => (
+      {tabTitles.map((title: any, index: number) => (
```
Comment (siva-kannan3) [inline src/modules/builder/editor/modules/basic/components/BasicHeader.tsx:16]:
```suggestion
      {tabTitles.map((title: string, index: number) => (
```

File: src/modules/builder/editor/modules/basic/BasicLayout.tsx (hunk 1 above)
Comment (sachinkumar579) [reply src/modules/builder/editor/modules/basic/BasicLayout.tsx]: This is done. Moved the line outside function

File: src/modules/builder/editor/modules/basic/components/BasicHeader.tsx (hunk 2 above)
Comment (sachinkumar579) [reply src/modules/builder/editor/modules/basic/components/BasicHeader.tsx]: Done.

File: src/modules/builder/editor/modules/basic/components/BasicHeader.tsx (hunk 3 above)
Comment (sachinkumar579) [reply src/modules/builder/editor/modules/basic/components/BasicHeader.tsx]: Done.

**Review by siva-kannan3 — APPROVED** [review]
LGTM

**Pull request merged**